src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,189 @@
+      <>
+        <div className={'fdsNavbar-mobileIconButton'}>
+          <IconButton
+            Icon={isOpen ? ActionsArrowUpIcon : ActionsArrowDownIcon}
+            onClick={() => handleClick(sectionName)}
+            label="dropdown"
+          />
+        </div>
+        <div className={'fdsNavbar-iconButton'}>
+          <ActionsArrowDownIcon />
+        </div>
+      </>
+    ),
+    [showClass]
+  );
+
+  return (
+    <header id="masthead" className="fdsNavbar-nav" role="navigation">
+      <div className={'fdsNavbar-topContainer'}>
+        <ul id="utility-menu" className={'fdsNavbar-utilityBar'}>
+          <li>
+            <a
+              href={`${site}/what-we-offer/technology-vendors/`}
+              className={'fdsNavbar-utilityBarLink'}
+            >
+              Technology Vendors
+            </a>
+          </li>
+          <li>
+            <a href={`${site}/research/`} className={'fdsNavbar-utilityBarLink'}>
+              Recent Research
+            </a>
+          </li>
+          <li>
+            <a href={`${site}/newsletter/`} className={'fdsNavbar-utilityBarLink'}>
+              Newsletter
+            </a>
+          </li>
+          <li>
+            <a href={`${site}/media-support/`} className={'fdsNavbar-utilityBarLink'}>
+              Media Support
+            </a>
+          </li>
+          <li>
+            <a href={`${site}/login`} className={'fdsNavbar-utilityBarLink'}>
+              Login
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div className={'fdsNavbar-container'}>
+        <div className={'fdsNavbar-siteBranding'}>
+          <a href={`${site}`}>
+            <img
+              className={'fdsNavbar-navLogo'}
+              width="168"
+              height="20"
+              alt="CB Insights Logo"
+              src={CBILogo}
+            />
+          </a>
+        </div>
+        <nav id="site-navigation" className={'fdsNavbar-navigation'}>
+          <div className={'fdsNavbar-navMenuButton'}>
+            <IconButton
+              size="m"
+              Icon={showClass.navList ? DenyIcon : HamburgerBarsIcon}
+              onClick={() => handleClick(NAV_LIST)}
+              theme="ghost"
+              label="Menu"
+            />
+          </div>
+          <div className={'fdsNavbar-navigationWrap'}>
+            <ul
+              id="site-menu"
+              className={cc([
+                { 'fdsNavbar-navShow': showClass.navList },
+                'fdsNavbar-navBar',
+              ])}
+            >
+              <li className={'fdsNavbar-navBarListItem'}>
+                <a className={'fdsNavbar-dropdownItemLink'}>Who We Serve</a>
+                <span className={'fdsNavbar-dropdownButton'}>
+                  {getArrowUpOrDown(showClass.whoWeServe, WHO_WE_SERVE)}
+                </span>
+                <NavBarDropdown
+                  content={WhoWeServeDropdown(site)}
```

Comment:
Convention in FDS is for components to use Pascal case, and for everything else to be camel case